src/common/__tests__/user-store.test.ts

```diff
@@ -47,30 +49,32 @@ describe("user store tests", () => {
     mockFs();
 
     di.override(writeFileInjectable, () => () => Promise.resolve());
-    di.override(directoryForUserDataInjectable, () => ({
-      get: () => "some-directory-for-user-data",
-    }));
+
     di.permitSideEffects(getConfigurationFileModelInjectable);
     di.permitSideEffects(userStoreInjectable);
 
-    di.override(releaseChannelInjectable, () => ({
-      get: () => "latest" as const,
-      init: async () => {},
-    }));
-
     di.unoverride(userStoreInjectable);
+
+    const runManySync = runManySyncFor(di);
+    const runAllBeforeElectronIsReady = runManySync(beforeElectronIsReadyInjectionToken);
+
+    runAllBeforeElectronIsReady();
```

Comment:
Strictly, this is a duplication of implementation details for what happens at start of application. It would be safer to make the test even more like "real application" by running this logic from the real application instead of reimagining it here. Stuff like this is likely to grow out of sync making the test unreliable and brittle.